### PR TITLE
DOC-33 - Updates to the graph viewer page

### PR DESCRIPTION
**Pull request by janettelynch**
Fix broken images

### CODE REVIEW

File: guides/quickstart-graph.md [hunk 1]
```diff
@@ -1,54 +1,59 @@
-# Navigating the JupiterOne Graphs
+# Navigating the JupiterOne Graphs-+
```
Comment (erkangz) [inline guides/quickstart-graph.md:1]: is the `-+` intentionally added?

**Review by AnneNielsenJ1 — APPROVED** [review]

File: guides/quickstart-graph.md [hunk 2]
```diff
@@ -1,54 +1,59 @@
...
-|     ![query-explan](../assets/icons/query-explan.png)     | Expand grouped nodes of the same type that have the same parent nodes. This option may not be always available depending on the data in the graph. |
-|   ![query-collapse](../assets/icons/query-collapse.png)   | Collapse nodes of the same type that have the same parent nodes into a group. This option may not be always available depending on the data in the graph. |
+| ![query-show-more](../assets/icons/query-show-more.png) | Display additional asset nodes that are directly connected to the selected asset but are not yet loaded into the graph. |
+| ![query-hide](../assets/icons/query-hide.png) | Hide the asset from the graph to reduce clutter. You can unhide all hidden assets from the bottom-left control. |
+| ![query-explan](../assets/icons/query-explan.png) | Expand grouped nodes of the same type that have the same parent nodes. This option may not be always available depending on the data in the graph. |
+| ![query-collapse](../assets/icons/query-collapse.png) | Collapse nodes of the same type that have the same parent nodes into a group. This option may not be always available depending on the data in the graph. |
```
Comment (erkangz) [inline guides/quickstart-graph.md:42]: You might want to consider a markdown formatter to make these look cleaner

Comment (erkangz) [reply guides/quickstart-graph.md:42]: And all the other tables.

Comment (janettelynch) [reply guides/quickstart-graph.md:42]: I'm working with Patrick to find these custom icons as they currently aren't in figma. I will replace them next week when I can get the files. I am working in an MD editor but I know the icons need replacing. Thanks

File: guides/quickstart-graph.md (hunk 1 above)
Comment (janettelynch) [reply guides/quickstart-graph.md]: Removed.

**Review by AnneNielsenJ1 — DISMISSED** [review]

**Review by AnneNielsenJ1 — APPROVED** [review]

**Pull request merged**